commands/upgrade_cluster.go

```diff
@@ -0,0 +1,311 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+	"sort"
+
+	"github.com/coreos/go-semver/semver"
+	"github.com/fatih/color"
+	"github.com/giantswarm/gsclientgen/models"
+	"github.com/spf13/cobra"
+
+	"github.com/giantswarm/gsctl/config"
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	// upgradeClusterActivityName assigns API requests to named activities
+	upgradeClusterActivityName = "upgrade-cluster"
+
+	upgradeDocsURL = "https://docs.giantswarm.io/reference/cluster-upgrades/"
```

Comment:
I remember you saying you wanted to drop the docs link for now. Don't remember the reasons.